pkg/apis/kops/dockerconfig.go

```diff
@@ -50,4 +50,7 @@ type DockerConfig struct {
 	Version *string `json:"version,omitempty"`
 	// LiveRestore enables live restore of docker when containers are still running
 	LiveRestore *bool `json:"liveRestore,omitempty" flag:"live-restore"`
+
+	// Hosts enables to configure custom hosts for the docker daemon to listen to
+	Hosts []string `json:"hosts,omitempty" flag:"host,repeat"`
```

Comment:
hi @jaipradeesh .. can you drop the newline and place ```Hosts``` alphabetical in the list please ..

Reply:
> // Hosts enables to configure custom hosts for the docker daemon to listen to

Looking at the rest of the items the appear what is does rather than what it allows you to do as such .. perhaps change to this to be more descriptive .. i.e. 

```// Hosts enables you to configure the endpoints the docker daemon listens on i.e tcp://0.0.0.0.2375 or unix:///var/run/docker.sock etc ```

Reply:
As @gambol99 mentioned examples are awesome

Reply:
Changes pushed.